.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+
+name: Build
+on:
+  push:
+    branches:
+    - '**'
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      matrix:
+        os: ["ubuntu-20.04", "windows-2019", [self-hosted, macOS, X64]]
+        # Pdfium is developed alongside Chromium and does not provide releases
+        # Upstream recommends using Chromium's dev channels instead
+        # Extract pdfium branch name used in stable channel from omahaproxy and do
+        # a checkout
+        # git checkout "chromium/$(curl https://omahaproxy.appspot.com/linux?channel=stable | cut -d'.' -f 3)"
+        # git checkout "chromium/$(curl https://omahaproxy.appspot.com/linux?channel=beta | cut -d'.' -f 3)"
+        # git checkout "chromium/$(curl https://omahaproxy.appspot.com/linux?channel=dev | cut -d'.' -f 3)"
+        chromium_version: ["chromium/4320"]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: build linux/macos
+        run: ./build.sh
+        if: runner.os != 'Windows'
+        env:
+          PDFium_BRANCH: ${{ matrix.chromium_version }}
```

Comment:
These environment variables could probably be  set globally for all jobs, to avoid duplicates